### ISSUE TITLE
Explain how to install the app

### ISSUE THREAD

**Issue by jms1voalte** [probot/settings#568]
## Problem Description

* In this repo's README.md file, under "Usage", item 1 is "Install the app", with a link to https://github.com/apps/settings

* On that page, under "Usage", is the same item 1, with the same link to https://github.com/apps/settings ... clicking that link looking for directions on how to install the app just takes you to the same page you're already on.

If there *is* a separate page explaining how to install the app, shouldn't these links point there instead of just pointing to the same page over and over again?

### What is actually happening

Links point to the same page that the link itself is on, without any details about how to "install" the app.

### What is the expected behavior

Links should point to a page which explains how to "install" the app.

**Comment by jms1voalte** [comment]
That's possible ... and if so, why doesn't the README.md file contain a link to THAT page? Having a link that says "Install the app", WHICH POINTS TO THE SAME PAGE, is rather pointless, no?

**Comment by jms1voalte** [comment]
**There is no green button that says "install" if you're not logged into a GitHub account.**

![image](https://user-images.githubusercontent.com/13367368/164278202-e7f93784-0ed7-4962-8c5c-3c36b81527a1.png)

In my case, I'm using Safari in "private mode", which means that the session cookies which make me "logged in" on one tab, don't carry over to new tabs. Once I logged in in the second tab I was able to see big green button, and things started to make sense.

So maybe it would make sense to add something which tells the user to log into their account if they don't see the big green button? (And included the link you provided, to a "how to install GitHub Apps in general" page, as well. That would be helpful to people like me for whom this is their first exposure to GitHub Apps.)

**Comment by foolioo** [comment]
@jms1voalte You are not signed-in to GitHub. Once you do, the green "install" button will appear.

**Comment by patcon** [comment]
@foolioo Agreed. @jms1voalte you should probably not expect to be finding a tailored install flow for an extension when you're not logged in. This feels quite niche, as the vast majority of people travelling this path will be logged in.

Having said that, speaking as a passerby here, I'm sure if you felt like submitting a small change to the PR to tip off someone like yourself in the future, the future person like yourself would be appreciative ❤️

**Comment by patcon** [comment]
Actually, just added it here, since it seems it's such a small fix that would make it clearer 🙏 
https://github.com/probot/settings/pull/573
